litmus-portal/frontend/src/utils/validate.ts

```diff
@@ -19,6 +19,15 @@ export const validateEmail = (value: string) => {
   return false;
 };
 
+export const validateUsername = (value: string) => {
+  const usernameValid = /^[a-zA-Z0-9]+$/;
```

Comment:
Username regex validation would require more checks like having special characters (Not all), - and @ are an important one.

Reply:
Hi @S-ayanide It is not emailed it is a username ,  as per the Issue The allowed characters are Alphabets, and Numeric